Tools/CISupport/ews-app/ews/common/github.py

```diff
@@ -213,7 +213,13 @@ class GitHubEWS(GitHub):
                           ['jsc-arm64', 'tv', 'mac-AS-debug-wk2', 'gtk-wk2', ''],
                           ['services', 'tv-sim', 'mac-wk2-stress', 'api-gtk', ''],
                           ['merge', 'watch', '', 'jsc-armv7', ''],
-                          ['unsafe-merge', 'watch-sim', '', 'jsc-armv7-tests', '']]
+                          ['unsafe-merge', 'watch-sim', '', 'jsc-armv7-tests', ''],
+                          ['', '', 'mac-ventura', '', ''],
```

Comment:
Each line here denotes one row in GitHub EWS status-bubble table.

So, all these new rows will have a single status-bubble, while there is space is rows above them. That would look somewhat ugly in the UI.
Some of these status-bubbles can be moved up in the table. For e.g.: 'mac-ventura' should be moved above in the line containing bubble for 'merge'. Similarly 'api-mac-ventura' and 'jsc-ventura-arm64' can be moved up. This would free up two additional rows.